eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -64,7 +64,7 @@ jobs:
 
       - script: |
           npm install ./eng/tools/versioning
-          node eng/tools/versioning/set-dev.js --build-id "$(Build.BuildNumber)" --repo-root "$(Build.SourcesDirectory)" --service "$(folder)"
+          node eng/tools/versioning/set-dev-dependencies.js --build-id "$(Build.BuildNumber)" --repo-root "$(Build.SourcesDirectory)" --service "$(folder)"
```

Comment:
Why a new script as opposed to updating the existing script?

Reply:
Just in case we decide to fall back again :P

Reply:
Copied back to `set-dev.js`